tests/backup/backup_helper.go

```diff
@@ -201,6 +204,30 @@ func CreateBackup(backupName string, clusterName string, bLocation string, bLoca
 	log.Infof("Backup [%s] created successfully", backupName)
 	return nil
 }
+
+func FilterAppContextsByNamespace(appContexts []*scheduler.Context, namespaces []string) (filteredAppContexts []*scheduler.Context) {
+	for _, appContext := range appContexts {
+		if Contains(namespaces, appContext.ScheduleOptions.Namespace) {
+			filteredAppContexts = append(filteredAppContexts, appContext)
+		}
+	}
+	return
+}
+
+// CreateBackupWithValidatation creates backup, checks for success, and validates the backup
+func CreateBackupWithValidatation(ctx context.Context, backupName string, clusterName string, bLocation string, bLocationUID string, scheduledAppContexts []*scheduler.Context, labelSelectors map[string]string, orgID string, uid string, preRuleName string, preRuleUid string, postRuleName string, postRuleUid string) error {
```

Comment:
nit: It should be `CreateBackupWithValidation`

Reply:
😅 fixed